.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  elixir: membraneframework/elixir@1
+  elixir: membraneframework/elixir@1.2
```

Comment:
This should not change, it allows using newer minor version without changing the config